web/env.html

```diff
@@ -82,6 +82,7 @@
         <form id="source" class="dropbox ui-layout-center"><textarea id="editor" placeholder="Type your code here..."></textarea></form>
 
         <div id="result" style="display:none" class="ui-layout-east">
+          <div id="obsolete-code-alert" class="obsolete-code-alert-fadeout"></div>
```

Comment:
Should there be text on this?  Something like: "Your code has been modified.  Click Run to update the  running program."